roles/realm_join/tasks/realm_join_ad.yml

```diff
@@ -0,0 +1,121 @@
+---
+- name: Ensure pexpect to answer password
+  become: yes
+  package:
+    name: pexpect
+    state: latest
```

Comment:
Does it really need to be `latest` (aka: trigger upgrade on re-run?)

Reply:
Probably not, but quite a new version is necessary, so I didn't want to go with just simple `present` in case there is old version, but no idea if that can even happen on 8 :thinking: